tst/integration/test_json_basic.py

```diff
@@ -287,6 +287,29 @@ def test_json_set_command_root_document(self):
                 'JSON.SET', key, '.', value)
             assert value.encode() == client.execute_command('JSON.GET', key)
 
+    def test_json_mset_command_root_document(self):
+        client = self.server.get_new_client()
+        # path to the root document is represented as '.'
+        assert b'OK' == client.execute_command(
+            "JSON.MSET",
+            k1, ".", '"Boston"',
+            k2, ".", '123',
+            k3, ".", '["Seattle","Boston"]',
```

Comment:
The paths are "root". Can you add additional tests that updates a child element?
e.g., for line 297, we can change it to:
```
json.mset ... k3 [1] '"NYC"'
```

In addition, make a more complicated json key with multiple levels. e.g.,
```
json.mset k1 . '{"firstName":"John","lastName":"Smith","age":27,"weight":135.25,"isAlive":true,"address":{"street":"21 2nd Street","city":"New York","state":"NY","zipcode":"10021-3100"},"phoneNumbers":[{"type":"home","number":"212 555-1234"},{"type":"office","number":"646 555-4567"}],"children":[],"spouse":null}'
```
Then, updating child elements:
```
json.mset k1 .age 30 k1 .address.city '"Boston"' k1 .phoneNumbers[0].type '"work"'
```